<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerWindow</class>
 <widget class="QDialog" name="registerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create User</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Imagenes/BookSeller-Logo.png</normalon>
   </iconset>
  </property>
  <widget class="QLabel" name="regisFondoLabel">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>321</width>
     <height>503</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/Imagenes/Bed Reading Spot Book Shelf Android Wallpaper.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="regisTituloLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kunstler Script</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Bukin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newUserLE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPassLE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>279</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newMailLE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>328</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Mail</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createPB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>400</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Create!</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
